### PR TITLE
Fresh runtimes.

### PR DESCRIPTION
**Pull request by AnturK**
One from the recent unicode changes the rest various nonsense.

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -576,7 +576,7 @@ GLOBAL_LIST_INIT(vox_sounds, list("abduction" = 'sound/vox_fem/abduction.ogg',
 "nine" = 'sound/vox_fem/nine.ogg',
 "nineteen" = 'sound/vox_fem/nineteen.ogg',
 "ninety" = 'sound/vox_fem/ninety.ogg',
-/datum/reagent/nitrogen = 'sound/vox_fem/nitrogen.ogg',
+"nitrogen" = 'sound/vox_fem/nitrogen.ogg',
```
Comment (kriskog) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Fucking chem ID's :grin:

File: code/__HELPERS/text.dm [hunk 2]
```diff
@@ -263,9 +263,11 @@
 	return trim_left(trim_right(text))
 
 //Returns a string with the first element of the string capitalized.
-/proc/capitalize(t as text)
-	. = t[1]
-	return uppertext(.) + copytext(t, 1 + length(.))
+/proc/capitalize(t)
+	. = t
```
Comment (kriskog) [inline code/__HELPERS/text.dm]: Will this ever serve a purpose?

Comment (TheVekter) [reply code/__HELPERS/text.dm]: Isn't that what's used to auto-cap the first letter of sentences when spoken IC?

Comment (AnturK) [reply code/__HELPERS/text.dm]: `""` vs `null`, i didn't go over the uses to figure out if any cares so just passed it on.

Comment (kriskog) [reply code/__HELPERS/text.dm]: Huh, fair enough

**Review by kriskog — APPROVED** [review]

**Comment by ShizCalev** [comment]
mind QDEL_NULL'ing main_part in destroy while you're at it?

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**